src/index.js

```diff
@@ -55,6 +55,19 @@ const expandShorthands = (tr) => {
 }
 
 /**
+ * Transforms __ref to ref
+ * @param {object} props
+ * @returns {object}
+ * @private
+ */
+const transformRef = ({ __ref, ...props }) => ({ ref: __ref, ...props })
```

Comment:
if `__ref` is not present, but `ref` is present — `ref` will be overwritten from `props`
it both of them are not present — still fine

or you just want to make it more obvious and a bit faster?